eng/pipelines/release-cli.yml

```diff
@@ -845,6 +879,24 @@ stages:
               workingDirectory: cli/installer/
             displayName: Test install script (cmd)
 
+          - pwsh: |
+              Get-ChildItem Cert:\LocalMachine\My | ForEach-Object {
+                if ($_.Subject -match "azd installer tests code signing") {
+                  Write-Host "Deleting $($_.PSPath) - $($_.Subject)"
+                  Remove-Item -Force $_.PSPath
+                }
+              }
+
+              Get-ChildItem Cert:\LocalMachine\Root | ForEach-Object {
+                if ($_.Subject -match "azd installer tests code signing") {
+                  Write-Host "Deleting $($_.PSPath) - $($_.Subject)"
+                  Remove-Item -Force $_.PSPath
+                }
+              }
+            displayName: Clean up self-signed certificates (Windows)
+            condition: contains(variables['Agent.OS'], 'Windows')
+            workingDirectory: hosting
```

Comment:
This could also go into a cleanup script.. something like `eng/scripts/Remove-AzdSelfSignedCert.ps1` or something.